synapse/lib/datapath.py

```diff
@@ -92,7 +92,7 @@ def _elem_iter(self):
         # of their names...
         if type( self._d_item ) == dict:
             for name,item in self._d_item.items():
-                yield initelem(item,name=name,parent=self)
+                yield initelem({0: name, 1: item},name=name,parent=self)
```

Comment:
thanks! my python is admittedly not the strongest... changing to a tuple also fixed the infinite recursion issue in '~foo' search routine that was breaking the build.